Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:8
+FROM node:10.16.0-alpine
```

Comment:
Did you test this Docker config? I've had some problems with alpine images in the past due to some packages requiring a more complete version of node.

Reply:
@imnotteixeira it looked fine, nothing un-usual detected. But if you could double check it, it would probabbly be best.

Reply:
Just for reference: The node version is exactly the same - only the underlying OS is much smaller. See [this here](https://github.com/alpinelinux/docker-alpine#why) for more information.

Reply:
I know, the problem was not with the node itself, but with some packages or dependencies that were required for some package installation

Reply:
Was just making it clear for anyone that might read the discussion 😅 

I'm guessing it would be something like `sass` or similar, I think some of these engines use `make` or `cmake` or stuff like that so they would most likely break, yes.